src/ADAL.PCL/Cache/TokenCacheKey.cs

```diff
@@ -68,17 +68,17 @@ internal TokenCacheKey(string authority, string resource, string clientId, Token
             this.DisplayableId = displayableId;
         }
 
-        public string Authority { get; private set; }
```

Comment:
I've made these properties read-only since they're used in the `GetHashCode` method.
Hash codes should _not_ be based on mutable fields.